README.md

```diff
@@ -4,12 +4,14 @@ This repository is mainly open to those who are looking to make some PR for the
 In this repository, you can find the solutions (as source code) for the problems in [HackerRank](https://hackerrank.com), [Codechef](https://codechef.com) or any other online platform and you can also add any coding projects.
 
 ### WHY SHOULD YOU CONTRIBUTE TO THIS REPOSITORY?
+
 - Beginner-friendly
 - Create your first Pull Request on GitHub
-- Start with any problem of your choice on [HackerRank](https://hackerrank.com), [Codechef](https://codechef.com), add programes file  etc
```

Comment:
Please make no further changes in the readme.md file to prevent potential conflict errors, and update your profile only at the end of contributors list. Thank you